tmt/steps/provision/testcloud.py

```diff
@@ -225,8 +288,8 @@ def show(
 
         super().show(keys=super_keys, verbose=verbose, logger=logger)
 
-        logger.info('memory', f"{self.memory} MB", 'green')
-        logger.info('disk', f"{self.disk} GB", 'green')
+        logger.info('memory', f"{self.memory or DEFAULT_MEMORY}", 'green')
+        logger.info('disk', f"{self.disk or DEFAULT_DISK} GB", 'green')
```

Comment:
```suggestion
        logger.info('disk', f"{self.disk or DEFAULT_DISK}", 'green')
```
The 'GB' should be probably deleted too.

Reply:
Removed in 570a950a